.github/workflows/lint-test-matrix.yaml

```diff
@@ -48,5 +49,10 @@ jobs:
         (steps.list-changed.outputs.changed == 'true') ||
         (contains(github.event.pull_request.labels.*.name, 'needs-testing'))
 
+    - name: Install metrics-server
+      id: metrics-server
+      run: |
+        helm upgrade --install --set args={--kubelet-insecure-tls} metrics-server metrics-server/metrics-server --namespace kube-system
+
```

Comment:
Do we actually need to install the metrics server? If it isn't installed the API will still accept the HPA objects, they just won't do anything.

Having the metrics server installed will slow down tests, may cause errors we don't actually care about if something goes wrong with the metrics server, and also may cause scaling to occur in Lagoon components in CI which will use more resources etc. I'm not sure if any of that is desirable?

Reply:
I've removed it now!